src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.Windows.cs

```diff
@@ -1,12 +1,31 @@
 ﻿using System;
 using System.Threading.Tasks;
+using Microsoft.Maui.DeviceTests.Stubs;
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Media;
+using Xunit;
 
 namespace Microsoft.Maui.DeviceTests
 {
 	public partial class LabelHandlerTests
 	{
+		[Fact(DisplayName = "Html Text Initializes Correctly")]
+		public async Task HtmlTextInitializesCorrectly()
+		{
+			var label = new LabelStub()
+			{
+				TextType = TextType.Html,
+				Text = "<h2><strong>Test1&nbsp;</strong>Test2</h2>"
+			};
+
+			var platformText = await GetValueAsync(label, (handler) =>
+			{
+				return handler.PlatformView.Text;
+			});
+
+			Assert.NotNull(platformText);
+		}
+
```

Comment:
Is it possible to move this test into the shared tests so all the platforms get this test? Also, not sure if it is possible, but maybe we could benefit with a test directly on `LabelHtmlHelper.ParseText` so that we can add to that as we find more cases and not need more device tests.

Reply:
Good suggestion. Moved the test to use is with all the platforms.